network/p2p/gossip/handler.go

```diff
@@ -54,9 +49,6 @@ type Handler[T Gossipable] struct {
 	set                Set[T]
 	metrics            Metrics
 	targetResponseSize int
-
-	pullLabels prometheus.Labels
-	pushLabels prometheus.Labels
```

Comment:
This was the source of the bug - only `pullLabels` was initialized in `NewHandler`.